config/install/rules.reaction.earn_points_for_voting_a_idea.yml

```diff
@@ -22,8 +22,25 @@ expression:
     conditions:
       -
         id: rules_condition
-        uuid: e42d9597-8b68-4c0f-8104-82cf94775eed
+        uuid: 64e03746-3bac-4912-8821-0f309c585a53
```

Comment:
Same here. How it's relevant to login form? :)

Reply:
Removed